LEGO1/omni/src/video/mxdisplaysurface.cpp

```diff
@@ -1114,8 +1116,8 @@ LPDIRECTDRAWSURFACE MxDisplaySurface::CreateCursorSurface()
 
 		newSurface->Unlock(ddsd.lpSurface);
 		DDCOLORKEY colorkey;
-		colorkey.dwColorSpaceHighValue = 31775;
-		colorkey.dwColorSpaceLowValue = 31775;
+		colorkey.dwColorSpaceHighValue = RGB555_CREATE(0x1f, 0, 0x1f);
```

Comment:
I think using 255 instead of 0x1f in these cases is easier to read/more common

Reply:
It's `31` actually (5 bits maximum): `0b11111` (but that's not supported by older compilers)
I chose hexadecimal representation as those are also used by css colors, and also shows that the number of bits is maximum 5.

Reply:
Ah yes of course. My bad.